Makefile

```diff
@@ -567,11 +569,13 @@ openstack_prep: export IMAGE=${OPENSTACK_IMG}
 openstack_prep: $(if $(findstring true,$(BMO_SETUP)), crc_bmo_setup) ## creates the files to install the operator using olm
 openstack_prep: $(if $(findstring true,$(NETWORK_BGP)), nmstate nncp netattach metallb metallb_config)
 openstack_prep: $(if $(findstring true,$(NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config)
+openstack_prep: $(if $(findstring true,$(INSTALL_CERT_MANAGER)) && $(findstring false,$(BMO_SETUP)), certmanager)
 	$(eval $(call vars,$@,openstack))
 	bash scripts/gen-olm.sh
 
 .PHONY: openstack
-openstack: certmanager operator_namespace openstack_prep ## installs the operator, also runs the prep step. Set OPENSTACK_IMG for custom image.
+openstack: openstack_prep operator_namespace
+openstack: ## installs the operator, also runs the prep step. Set OPENSTACK_IMG for custom image.
```

Comment:
Why do you have a separate rule without any pre-requisites?

Reply:
All the pre-requisities will be provided and configured by ci-framework role. The goal of this pr is to make sure we only generate openstack olm files and deploy openstack operator. This will help us to testing validated architecture crs.

Reply:
Sorry you mean about line 578 changes?

Reply:
What I'm saying is L578 is not requred. You can remove it.